### ISSUE TITLE
pop-up window is in the wrong spot using i3wm

### ISSUE THREAD

**Issue by PeterMitrano** [kiteco/issue-tracker#309]
**Required Information**
To help us triage your issue effectively, please make sure you've completed these steps before opening this issue. Issues with not enough information may be deprioritized or ignored.

https://s3.console.aws.amazon.com/s3/object/kite-client-logs/prod/425058/kite_local/client.log.2020-08-04_09-36-09-PM.bak.gz
Using PyCharm plugin 1.6.7
Kite Version: 2.20200728.0
- [ ] If reporting an issue with Kite's editor plugin, include which editor you are using and the version, and any accompanying logs from your editor's console.

**Describe the bug**
The completion dialog is in the wrong spot, no matter where I am typing it appears at the top of the screen.

**Screenshots**
![2020_08_04-21-35-55](https://user-images.githubusercontent.com/4010770/89361997-b8165380-d69a-11ea-877f-fafcde0b9559.png)

**Comment by its-dhung** [comment]
@PeterMitrano I assume this only started happening after you installed the Kite plugin?

**Comment by PeterMitrano** [comment]
Yes, the normal pycharm autocomplete dialogue appears in the correct spot as I type.
